internal/operators/common/common.go

```diff
@@ -0,0 +1,32 @@
+package common
```

Comment:
Maybe a little too nit: but not sure if the name `common` is ok since the `internal/common` pkg exists already 😅 @avishayt @carbonin wdyt?

Reply:
I totally agree that common is a crappy name, I've maintained it for consistency as it is used throughout the code.
Adding this to `operators` package would've caused circular dependency as `operators` package is requiring `operators/odf` and `operators/odf` would require this function(s).

As I discussed in another comment, I think the best approach to this would be using `<package>utils` pattern (see for example std lib `ioutils`). This way we can segregate in a package all related utilities and minimize circular dependency issues.
For example, all functions (not methods) related cluster object could go in `internal/clusterutils`. This would reduce dependencies to a minimum and can be used from the whole codebase without any issue.